website/build.gradle

```diff
@@ -91,30 +92,38 @@ task startDockerContainer(type: Exec) {
     "${->createDockerContainer.containerId()}" // Lazily evaluate containerId.
 }
 
+task initGitSubmodules(type: Exec) {
+    commandLine 'docker', 'exec',
+                "${->startDockerContainer.containerId()}", 'git',  'submodule', 'update', '--init',  '--recursive'
+}
+
+task installDependencies(type: Exec) {
+    commandLine 'docker', 'exec', '--workdir', "$dockerSourceDir",
+                "${->startDockerContainer.containerId()}", 'yarn', 'install'
+}
+
+task buildGithubSamples(type: Exec) {
+  commandLine 'docker', 'exec', '--workdir', "$dockerSourceDir",
+              "${->startDockerContainer.containerId()}", 'yarn', 'build_github_samples'
```

Comment:
What is build_github_samples ?

Reply:
I think I understand this. Does it move snippets to files?

Comments about these tasks will help.